src/partners/exolix.ts

```diff
@@ -66,29 +77,29 @@ const statusMap: { [key in ExolixStatus]: Status } = {
   sending: 'other'
 }
 
+type Response = ReturnType<typeof fetch>
+
 export async function queryExolix(
   pluginParams: PluginParams
 ): Promise<PluginResult> {
-  const ssFormatTxs: StandardTx[] = []
-  let apiKey: string
-  let latestTimestamp = 0
-  if (typeof pluginParams.settings.latestTimestamp === 'number') {
-    latestTimestamp = pluginParams.settings.latestTimestamp
-  }
+  const { settings, apiKeys } = asExolixPluginParams(pluginParams)
+  const { apiKey } = apiKeys
+  let { latestIsoDate } = settings
 
-  if (typeof pluginParams.apiKeys.apiKey === 'string') {
-    apiKey = pluginParams.apiKeys.apiKey
-  } else {
-    return {
-      settings: { latestTimestamp },
-      transactions: []
-    }
+  if (typeof apiKey !== 'string') {
```

Comment:
The cleaner has already limited this to `string | undefined`, so it would be simpler to do `apiKey == null` instead of using `typeof`, since `typeof` seems to suggest that the data is un-typed (but it isn't anymore).